# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dovecot/dovecot-2.2-0002-fc9ef7d2251a.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---


# HG changeset patch
# User Timo Sirainen <tss@iki.fi>
# Date 1365964049 -10800
# Node ID fc9ef7d2251aec477d73bacfec82c0f66cc813b3
# Parent  a7d0bb66639f956329916020ca6717b8840f842d
lib-storage: mailbox_list_index=yes was still broken.

diff -r a7d0bb66639f -r fc9ef7d2251a src/lib-storage/mail-namespace.c
--- a/src/lib-storage/mail-namespace.c	Sun Apr 14 20:40:47 2013 +0300
+++ b/src/lib-storage/mail-namespace.c	Sun Apr 14 21:27:29 2013 +0300
@@ -467,6 +467,7 @@
 	user->namespaces = ns;
 
 	T_BEGIN {
+		hook_mail_namespaces_added(ns);
 		hook_mail_namespaces_created(ns);
 	} T_END;
 	return 0;

